packages/libretro/vbam/package.mk

```diff
@@ -19,16 +19,16 @@
 ################################################################################
 
 PKG_NAME="vbam"
-PKG_VERSION="36b011c"
+PKG_VERSION="3ee26c0"
 PKG_REV="1"
 PKG_ARCH="any"
 PKG_LICENSE="GPLv2"
-PKG_SITE="https://github.com/libretro/vbam-libretro"
+PKG_SITE="https://github.com/visualboyadvance-m/visualboyadvance-m"
```

Comment:
Buildbot uses the libretro repo:
https://github.com/libretro/libretro-super/blob/master/recipes/linux/cores-linux-x64-generic#L137

I'd prefer we target the same repos as buildbot.

Reply:
We are not core developers but core integraters and we pick what is best for our users.
It's not us that need to switch but the buildbot.